pyproject.toml

```diff
@@ -18,9 +18,9 @@ classifiers = [
 dependencies = [
     "cached_path>=1.6.2",
     "ase>=3.23.0",
-    "numpy>=1.26.4, <2.0.0",
+    "numpy>=1.26.4",
     "scipy>=1.13.1",
-    "torch==2.2.0",
```

Comment:
Can you add major version bounds above? e.g we do not want to get to torch 3.0 and assume that it will be backward compatible.

Reply:
yep